src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.html

```diff
@@ -0,0 +1,112 @@
+        <img *ngIf="!order.extend" src="assets/img/ubs-admin-orders/arrow_up.svg" alt="arrow up" (click)="changeCard(order.id)" />
+        <div>{{ order.order }}</div>
+        <div>{{ order.status }}</div>
+        <div class="orderSum">{{ order.sum }}</div>
+        <div class="btnsGroup">
+          <div *ngIf="order.status === 'Сформовано'" class="btnGridCancel"><button class="btnCancel">Скасувати</button></div>
+          <div *ngIf="order.status === 'Виконано/Неоплачено'" class="btnGridFake"><button class="btnFake"></button></div>
+          <div *ngIf="order.status === 'Сформовано' || order.status === 'Виконано/Неоплачено'" class="btnGridPay">
+            <button class="btnPay">Оплатити</button>
+          </div>
+          <div *ngIf="order.status === 'На маршуті' || order.status === 'Підтверджено' || order.status === 'Виконано'" class="btnGridOrder">
+            <button class="btnOrder">Замовити ще раз</button>
+          </div>
+        </div>
+      </div>
+      <div *ngIf="order.extend" class="fullCard">
+        <div class="headerDetails">Деталі замовлення</div>
+        <table class="tableOfDetails">
+          <tr class="headerTable">
+            <th>Послуги</th>
+            <th>Об'єм</th>
+            <th>Вартість</th>
+            <th>Кількість пакетів</th>
+            <th>Сума</th>
+          </tr>
+          <tr class="dataTable">
+            <td>Вторсировина</td>
+            <td>120 л</td>
+            <td>200 грн</td>
+            <td>1 шт</td>
+            <td>200 грн</td>
+          </tr>
+          <tr class="dataTable">
+            <td>Безнадійний одяг</td>
+            <td>120 л</td>
+            <td>300 грн</td>
+            <td>1 шт</td>
+            <td>300 грн</td>
+          </tr>
+          <tr class="dataTable">
+            <td>Безнадійний одяг</td>
+            <td>20 л</td>
+            <td>50 грн</td>
+            <td>2 шт</td>
+            <td>100 грн</td>
+          </tr>
+          <tr>
+            <td></td>
+          </tr>
+          <tr class="sumOfOrder">
+            <td colspan="4">Сума замовлення:</td>
+            <td>500 грн</td>
+          </tr>
+          <tr *ngIf="order.status !== 'Виконано/Неоплачено'" class="optionalRow">
+            <td colspan="4">Сертифікат:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr *ngIf="order.status !== 'Виконано/Неоплачено'" class="optionalRow">
+            <td colspan="4">Бонуси:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr *ngIf="order.status === 'Виконано/Неоплачено'" class="optionalRow">
+            <td colspan="4">Сума наданих послуг:</td>
+            <td>-100 грн</td>
+          </tr>
+          <tr class="sumToPay">
+            <td colspan="4">Сума до оплати:</td>
+            <td>350 грн</td>
+          </tr>
+        </table>
+      </div>
+      <div *ngIf="order.extend" class="orderDetails">
+        <ol style="list-style-type: none">
```

Comment:
move this to class
also maybe use Bootstrap

Reply:
Moved this property to the scss file.